src/Typewriter/Options.cs

```diff
@@ -81,5 +81,8 @@ public class Options
         [Option('t', "transform", HelpText = "Specify the URI to the XSLT that will preprocess the metadata. Overrides the" +
             "cleaning done by embeddeded typewriter.exe rules.")]
         public string Transform { get; set; }
+
+        [Option('r', "removeannotations", Default = (bool)true, HelpText = "Specify whether to keep source CSDL capability annotations. Only applicable when using generationMode Transform or TransformWithDocs.")]
```

Comment:
```suggestion
        [Option('r', "removeannotations", Default = true, HelpText = "Specify whether to keep source CSDL capability annotations. Only applicable when using generationMode Transform or TransformWithDocs.")]
```

Reply:
It is a requirement that we unbox when we have a default boolean value.

https://github.com/commandlineparser/commandline/wiki/Option-Attribute#default